src/app/navbar/navbar.component.html

```diff
@@ -43,6 +43,14 @@
             <span class="fas fa-plus-circle"></span> Create Proposal
           </a>
         </div>
+         <div class="dropdown-menu" aria-labelledby="playDropdown" ngbDropdownMenu>
```

Comment:
This menu should be appropriately formatted to be displayed. 

Moreover, it is better to organise all Invite-related actions into an "Invite" dropdown, which might then include submenus like List or Create